README.md

```diff
@@ -132,7 +134,13 @@ $ docker build -f ci/Dockerfile -t ternd .
 +ENTRYPOINT ["tern", "-q"]
 ```
 
-Run the script `docker_run.sh`. You may need to use sudo. In the below command `debian` is the docker hub container image name  and `buster` is the tag that identifies the version we are interested in analyzing.
+Run the ternd container image
+
+```
+$ docker run --rm ternd report -i debian:buster
```

Comment:
We need to add skopeo as a dependency to the Dockerfile located at `docker/Dockerfile`. The README points users to this file to build `ternd`.

Even with that change, though, I can't get this to work on my mac or on my linux machine? Is it working for you? I get a very long trackback error in both cases ending with:

```
  File "/usr/local/lib/python3.9/site-packages/tern/__main__.py", line 287, in main
    do_main(args)
  File "/usr/local/lib/python3.9/site-packages/tern/__main__.py", line 116, in do_main
    crun.execute_image(args)
  File "/usr/local/lib/python3.9/site-packages/tern/analyze/default/container/run.py", line 75, in execute_image
    image_string = extract_image(args)
  File "/usr/local/lib/python3.9/site-packages/tern/analyze/default/container/run.py", line 32, in extract_image
    image_attrs = docker_api.dump_docker_image(args.docker_image)
  File "/usr/local/lib/python3.9/site-packages/tern/load/docker_api.py", line 208, in dump_docker_image
    client = check_docker_setup()
  File "/usr/local/lib/python3.9/site-packages/tern/load/docker_api.py", line 31, in check_docker_setup
    client = docker.from_env(timeout=180)
  File "/usr/local/lib/python3.9/site-packages/docker/client.py", line 96, in from_env
    return cls(
  File "/usr/local/lib/python3.9/site-packages/docker/client.py", line 45, in __init__
    self.api = APIClient(*args, **kwargs)
  File "/usr/local/lib/python3.9/site-packages/docker/api/client.py", line 197, in __init__
    self._version = self._retrieve_server_version()
  File "/usr/local/lib/python3.9/site-packages/docker/api/client.py", line 221, in _retrieve_server_version
    raise DockerException(
docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))
```

Reply:
Will check it out.

Reply:
Oh I know why. It's because the tern version installed in this image is from pypi which _doesn't_ have these changes. So you still need to run this with the `docker_run` script to make it work. This should work after a release, but I don't see how to resolve this in the meantime.

Reply:
@nishakm Ok, let me re-test with the ci dockerfile. We still need to add skopeo as a dependency to docker/Dockerfile.

Reply:
Alright, confirmed that the issue is me using the Dockerfile that doesn't have the changes (*face palm*) sorry about that. It works with the ci Dockerfile :)